whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/config/NrtmClientTransactionConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package net.ripe.db.nrtm4.client.config;
+
+import net.ripe.db.nrtm4.client.condition.Nrtm4ClientCondition;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.AdviceMode;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+
+@Configuration
+@Conditional(Nrtm4ClientCondition.class)
+@EnableTransactionManagement(mode = AdviceMode.ASPECTJ)
+public class NrtmClientTransactionConfiguration {
+
+    public static final String NRTM_CLIENT_UPDATE_TRANSACTION = "nrtm-client-update-transaction-manager";
```

Comment:
No need to define fields if value is only used once

Reply:
It is public, it is being in use by repository layer...
Is how TransactionConfiguration works as well